### PR TITLE
Reformatted 10 recipes

### PR DESCRIPTION
**Pull request by dash102**
https://codein.withgoogle.com/dashboard/task-instances/4796400818716672/?sp-page=1

### CODE REVIEW

**Comment by waddlesplash** [comment]
@dash102 What are you doing with #359 now?

File: haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe [hunk 1]
```diff
@@ -33,8 +31,14 @@ REQUIRES="
...
 	"
 
-BUILD_REQUIRES="
-	haiku${secondaryArchSuffix}_devel
+PROVIDES_devel="haikuwebkit${secondaryArchSuffix}_devel = $portVersion
+	devel:libjavascriptcore$secondaryArchSuffix = $portVersion
+	devel:libwebcore$secondaryArchSuffix = $portVersion
+	devel:libwebkit$secondaryArchSuffix = $portVersion
+	devel:libwtf$secondaryArchSuffix = $portVersion"
+REQUIRES_devel="haikuwebkit$secondaryArchSuffix == $portVersion base"
```
Comment (waddlesplash) [inline haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe]: All PROVIDES*/*REQUIRES*/*FILES* get the extra line at the beginning and end, as I said before. So this should be 
```
PROVIDES_devel="
	haikuwebkit${secondaryArchSuffix}_devel = $portVersion
	devel:libjavascriptcore$secondaryArchSuffix = $portVersion
	devel:libwebcore$secondaryArchSuffix = $portVersion
	devel:libwebkit$secondaryArchSuffix = $portVersion
	devel:libwtf$secondaryArchSuffix = $portVersion
	"
```
Please fix these.

**Comment by dash102** [comment]
Oops sorry, I misunderstood about the extra lines. Regarding #359, I'm not really sure what to do with it - is it possible to delete it?

**Comment by dash102** [comment]
How do I edit and resubmit the recipe files?

**Comment by waddlesplash** [comment]
You must do it from your local Git clone (`git commit --amend`). Also please squash your commits at the same time.

**Comment by dash102** [comment]
I edited the files online through github.com, not the terminal, and when I use <git commit --amend> in the terminal, I get a "fatal: Not a git repository" error.

**Comment by waddlesplash** [comment]
Did you clone the repository first?

**Comment by dash102** [comment]
I forked the repository, not sure if that's the same thing.

**Pull request closed without merging**